test/integration/homepage_test.rb

```diff
@@ -26,4 +26,28 @@ class HomepageTest < ActionDispatch::IntegrationTest
     assert page.has_content? "Agile delivery How to work in an agile way: principles, tools and governance."
     assert page.has_link? 'Agile delivery', href: '/service-manual/agile-delivery'
   end
+
+  test 'the homepage includes a link to the digital service standard' do
+    setup_and_visit_example('service_manual_homepage', 'service_manual_homepage')
+
+    assert page.has_content? <<-TEXT
+      The Digital Service Standard provides the principles of building a good
+      digital service. This manual explains what teams can do to build great
+      digital services that will meet the standard.
+    TEXT
+
+    assert page.has_link? 'Digital Service Standard', href: '/service-manual/service-standard'
+  end
+
+  test 'the homepage includes a link to the communities of practise' do
+    setup_and_visit_example('service_manual_homepage', 'service_manual_homepage')
+
+    assert page.has_content? <<-TEXT
+      You can view the communities of practice to find more learning
+      resources, see who has written the guidance in the manual and connect
+      with digital people like you from across government.
+    TEXT
+
+    assert page.has_link? 'communities of practice', href: '/service-manual/communities'
+  end
```

Comment:
I think so. That's not to say that the text won't change before then. Do you think the descriptions should be removed? My only concern is that the Digital Service Standard link already exists on the page, so the test for that would pass even if this addition didn't exist. Unless I use a `within`, or we just sack these tests off all together.